packages/fxa-auth-server/lib/senders/emails/templates/postAddAccountRecovery/index.mjml

```diff
@@ -0,0 +1,45 @@
+<%# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/. %>
+
+<mj-include path="./css/global.css" type="css" css-inline="inline" />
+
+<mj-section>
+  <mj-column>
+    <mj-text css-class="text-header">
+      <span data-l10n-id="postAddAccountRecovery-title">Account recovery key generated</span>
+    </mj-text>
+
+    <mj-text css-class="text-body">
+      <span data-l10n-id="postAddAccountRecovery-description">You have successfully generated an account recovery key for your Firefox Account using the following device:</span>
+    </mj-text>
+  </mj-column>
+</mj-section>
+
+<%- include('/partials/location/index.mjml') %>
+
+<mj-section>
+  <mj-column>
+    <mj-text css-class="text-body">
+      <span data-l10n-id="postAddAccountRecovery-recovery">
+        If this was not you,
+        <a class="link-blue" href="<%- revokeAccountRecoveryLink %>" data-l10n-name="revokeAccountRecoveryLink">click here</a>.
+      </span>
+    </mj-text>
+  </mj-column>
+</mj-section>
+
+<%- include('/partials/button/index.mjml') %>
```

Comment:
Ooo it looks like the button should link  out to `/settings` per the old email but it's linking out to `/account_recovery` here

Reply:
Hmm, I'm not seeing that. The "If this was not you, click here." text is linking to `/settings/account_recovery` and the "Manage account" button is linking to `/settings`, which is consistent with the old template.

Reply:
I see where the hang up is. I only checked this in Storybook, where "Manage account" is linking to `/account_recovery`:

<img width="806" alt="image" src="https://user-images.githubusercontent.com/13018240/133296656-ad237115-5224-496f-9d54-b25da24e0c52.png">

... but I just tried checking this after running `write-emails` for the new templates, and I'm seeing it render as expected.

You just need to update `link` in `postAddAccountRecovery` stories.